pysatSpaceWeather/instruments/methods/kp_ap.py

```diff
@@ -212,16 +247,15 @@ def calc_daily_Ap(ap_inst, ap_name='3hr_ap', daily_name='Ap',
     if running_name is not None:
         ap_inst[running_name] = ap_mean
 
-        meta_dict = {ap_inst.meta.labels.name: running_name,
-                     ap_inst.meta.labels.desc: "running daily Ap index",
-                     ap_inst.meta.labels.min_val: 0,
-                     ap_inst.meta.labels.max_val: 400,
-                     ap_inst.meta.labels.fill_val:
-                     ap_inst.meta[ap_name][ap_inst.meta.labels.fill_val],
-                     ap_inst.meta.labels.notes:
-                     '24-h running average of 3-hourly ap indices'}
-
-        ap_inst.meta[running_name] = meta_dict
+        initialize_ap_metadata(ap_inst.meta, running_name,
+                               ap_inst.meta[ap_name][
+                                   ap_inst.meta.labels.fill_val])
+        ap_inst.meta[running_name,
+                     ap_inst.meta.labels.desc] = "running daily Ap index"
+        ap_inst.meta[running_name,
+                     ap_inst.meta.labels.notes] = ''.join(['24-h running ave',
```

Comment:
```suggestion
                     ap_inst.meta.labels.notes] = ''.join(['24-h running average',
```

Reply:
I shortened this for the line length.  How much to you not want average abbreviated?